cli/src/main/scala/scalaxb/compiler/Main.scala

```diff
@@ -130,6 +130,9 @@ object Arguments {
         c.remove(VarArg) }
       opt[Unit]("ignore-unknown") text("ignores unknown Elements") action { (_, c) =>
         c.update(IgnoreUnknown) }
+      opt[Unit]("discard-non-identifiers") text("Discards any characters that are invalid in Scala identifiers such as dots and hyphens") action { (_, c) =>
```

Comment:
We discussed this on hangout, and agreed on something like `symbol-strategy ignore`.
I guess the discard false could be called `symbol-strategy 1.0`.